src/views/Fulfillment.vue

```diff
@@ -78,7 +73,7 @@
             </ion-item>
             <ion-item lines="none">
               <ion-label class="ion-text-wrap">
-                <p>{{ translate("Sends notifications for open, ready-for-pickup BOPIS orders and brokered, packed, and completed Sales orders.") }}</p>
+                <p>{{ translate("Sends notifications for open, ready-for-pickup BOPIS orders, packed, and completed Sales orders.") }}</p>
```

Comment:
This change is not needed as the above job sends notification for brokered orders as well.